catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/BrickValueParameterTest.java

```diff
@@ -670,17 +670,17 @@ public void testDataBricksDefaultValues() {
 		checkIfBrickShowsText(InsertItemIntoUserListBrick.class, R.string.brick_insert_item_into_userlist_insert_into);
 		checkIfBrickShowsText(InsertItemIntoUserListBrick.class, R.string.brick_insert_item_into_userlist_into_list);
 		checkIfBrickShowsText(InsertItemIntoUserListBrick.class, R.string.brick_insert_item_into_userlist_at_position);
-		checkIfBrickShowsText(InsertItemIntoUserListBrick.class, "1.0");
-		checkIfBrickShowsText(InsertItemIntoUserListBrick.class, "1");
+		checkIfBrickShowsEditTextWithText(InsertItemIntoUserListBrick.class, R.id.brick_insert_item_into_userlist_value_edit_text, "1");
+		checkIfBrickShowsEditTextWithText(InsertItemIntoUserListBrick.class, R.id.brick_insert_item_into_userlist_at_index_edit_text, "1");
 		checkIfBrickShowsSpinnerWithEditTextOverlayWithText(InsertItemIntoUserListBrick.class,
 				R.id.insert_item_into_userlist_spinner,
 				R.string.brick_variable_spinner_create_new_variable);
 
 		checkIfBrickShowsText(ReplaceItemInUserListBrick.class, R.string.brick_replace_item_in_userlist_replace_in_list);
 		checkIfBrickShowsText(ReplaceItemInUserListBrick.class, R.string.brick_replace_item_in_userlist_item_at_index);
 		checkIfBrickShowsText(ReplaceItemInUserListBrick.class, R.string.brick_replace_item_in_userlist_with_value);
-		checkIfBrickShowsText(ReplaceItemInUserListBrick.class, "1.0");
-		checkIfBrickShowsText(ReplaceItemInUserListBrick.class, "1");
+		checkIfBrickShowsEditTextWithText(ReplaceItemInUserListBrick.class, R.id.brick_replace_item_in_userlist_at_index_edit_text, "1");
+		checkIfBrickShowsEditTextWithText(ReplaceItemInUserListBrick.class, R.id.brick_replace_item_in_userlist_value_edit_text, "1");
```

Comment:
@84n4n4, please take a look at these method changes. I'm definitely not up-to-date with espresso tests.
If they are fine, feel free to merge this PR :wink:

Reply:
Hey, I can quickly say why I changed it that way.
The checkIfBrickShowsText expects that there is exactly one field of the Brick with the expected value, but because i got rid of the decimals there are now two fields with the same value, therefore an error gets raised and fails the test. The method I use now also checks if the text is the same, but I need to specify the field which the text gets compared to eg. R.id.brick_replace_item_in_userlist_at_index_edit_text.